backend/app/tests/validators/test_lexical_slurs.py

```
@@ -0,0 +1,140 @@
+    }[severity]
+
+    return LexicalSlur(
+        severity=sev,
+        languages=languages or ["en", "hi"]
+    )
+
+# ---------------------------------------
+# TESTS
+# ---------------------------------------
+
+def test_passes_when_no_slur(patch_slur_load):
```

Ruff (0.14.10)

Unused function argument: `patch_slur_load`

(ARG001)